files/.prettierignore

```diff
@@ -1,19 +1,10 @@
 # Prettier is also run from each package, so the ignores here
 # protect against files that may not be within a package
 
-# dependencies
-/node_modules/
-
 # misc
 !.*
-.eslintcache
```

Comment:
`.eslintcache` is not a file that Prettier recognises, and thus picks up by default.